pkg/util/file_indexer.go

```
@@ -26,14 +27,10 @@ func read(filePath string) (map[string]FileData, error) {
 		return map[string]FileData{}, nil
 	}
 
-	jsonFileRead, err := os.Open(filePath)
-	// if we os.Open returns an error then handle it
+	byteValue, err := ioutil.ReadFile(filePath)
 	if err != nil {
 		return nil, err
 	}
-	defer jsonFileRead.Close()
-
-	byteValue, _ := ioutil.ReadAll(jsonFileRead)
```

fix: ignored the error here